src/inmanta/agent/agent.py

```diff
@@ -1360,17 +1360,6 @@ async def ensure_code(self, environment: uuid.UUID, version: int, resource_types
                             )
                             requirements.update(source["requirements"])
 
-                            module_name: str = loader.get_inmanta_module_name(source["module_name"])
-                            all_installed_modules.add(module_name)
-
-                        all_required_modules.update(
```

Comment:
I think `all_required_modules` and `all_installed_modules` are no longer used and can be removed https://github.com/inmanta/inmanta-core/blob/1f4d680a2398b997eb244f290c0038ac444237eb/src/inmanta/agent/agent.py#L1328-L1331